Classes/KIFTestActor.m

```diff
@@ -179,7 +180,13 @@ - (void)waitForTimeInterval:(NSTimeInterval)timeInterval
 }
 
 -(void)waitForAnimationsToFinish {
-    NSTimeInterval maximumWaitingTimeInterval = 3;
+    NSTimeInterval maximumWaitingTimeInterval = self.animationWaitingTimeout;
+    if(maximumWaitingTimeInterval <= 0) {
+        return;
+    }
+    
+    // Wait for the view to stabilize and give them a chance to start animations before we wait for them.
+    [self waitForTimeInterval:0.5f];
     
     NSTimeInterval startTime = [NSDate timeIntervalSinceReferenceDate];
     [self runBlock:^KIFTestStepResult(NSError **error) {
```

Comment:
This is more style, so it obviously isn't absolutely necessary. It just makes the code a bit easier to read and the logic a bit simpler in my opinion. Maybe something for @Sipeso to keep in mind in the future.